DFC.ServiceTaxonomy.Editor/Recipes/ft2-82-content-approval-module.recipe.json

```diff
@@ -0,0 +1,99 @@
+{
+    "name": "01_content_approval",
+    "displayName": "Content approval",
+    "description": "Enables content approval with associated roles and permissions",
+    "author": "National Careers Service",
+    "website": "https://github.com/SkillsFundingAgency/dfc-servicetaxonomy-editor",
+    "version": "1.0",
+    "issetuprecipe": false,
+    "categories": ["default"],
+    "tags": ["ncs"],
+    "steps": [
+        {
+            "name": "feature",
+            "disable": [],
+            "enable": [
+                "DFC.ServiceTaxonomy.ContentApproval"
+            ]
+        },
+        {
+            "name": "Roles",
+            "Roles": [
+                {
+                    "Name": "ContentAdministrator",
+                    "Description": "Content administrator",
+                    "Permissions": [
+                        "ListContent",
+                        "DeleteContent",
+                        "EditContent",
+                        "CloneContent",
+                        "ViewContent",
+                        "PreviewContent",
+                        "PublishContent",
+                        "CanPerformContentDesignReviewPermission",
+                        "CanPerformSMEReviewPermission",
+                        "CanPerformUXReviewPermission",
+                        "CanPerformStakeholderApprovalPermission",
+                        "AssignRoles",
+                        "ManageRoles",
+                        "ManageTaxonomy",
+                        "ManageUsers",
+                        "ManageOwnUserInformation"
+                    ]
+                },
+                {
+                    "Name": "Editor",
+                    "Description": "Editor",
+                    "Permissions": [
+                        "ListContent",
+                        "EditContent",
+                        "ViewContent",
+                        "PreviewContent",
+                        "CloneContent",
+                        "PublishContent"
+                    ]
```

Comment:
I suspect we might have to allow the editor to have `ManageTaxonomy` permissions, so that they can edit job profile page locations.